lib/health-indicator/index.ts

```diff
@@ -1,5 +1,6 @@
 export * from './health-indicator-result.interface';
 export * from './health-indicator';
+export { HealthIndicatorService } from './health-indicator.service';
```

Comment:
Could you please also export `HealthIndicatorSession` (`*`)?
We would like to use the type `HealthIndicatorSession` in our tests.
Currently we need to use `import { HealthIndicatorSession } from '@nestjs/terminus/dist/health-indicator/health-indicator.service';`.

Reply:
@SchroederSteffen I'd prefer to keep it that way unless it's really inconvenient. 
I'd suggest using that instead:

```typecript
type HealthIndicatorSession = ReturnType<HealthIndicatorService['check']>
```